runtimes/spiritnet/src/filter.rs

```diff
@@ -0,0 +1,68 @@
+use frame_support::traits::Contains;
+
+use super::Call;
+
+pub struct GovCalls;
+impl Contains<Call> for GovCalls {
+	fn contains(t: &Call) -> bool {
+		return matches!(
+			t,
+			Call::Democracy(..)
+				| Call::Council(..)
```

Comment:
M: If we want to use Council origin for switching filters on and off, this line has to be excluded.

Reply:
Yeah I think we should make sure that 
1. `set_filter` is always callable
2. the council (or who ever is allowed to set the filter) can do their work to set the filter